### ISSUE TITLE
Accessibility issue - Pagination Ellipses Lack Accessible Name (WCAG 4.1.2 – Level A)

### ISSUE THREAD

**Issue by sherenemorris03** [DFE-Digital/find-information-products-services-frontend#115]
**Describe the bug**
Issue 7 in the Accessibility report
The ellipses used in pagination do not have an accessible name. This means screen reader users cannot determine what the control does or where it will take them, leading to confusion and potential navigation errors.

User story:
As a screen reader user,
I want to understand the purpose of all pagination controls,
So that I can navigate through pages confidently and without confusion.

**To Reproduce**
Steps to reproduce the behaviour:
1. Navigate to the Products page.
2. Use a screen reader to explore the pagination controls.
3. Observe that the ellipses lack descriptive text.

**Expected behaviour**
All pagination controls should have accessible names that describe their function.
The ellipses are announced without context or description.

**Screenshots**

<img width="945" height="105" alt="Image" src="https://github.com/user-attachments/assets/2dd1cc5c-1632-43a8-8099-bab52abd69f9" />

<img width="946" height="71" alt="Image" src="https://github.com/user-attachments/assets/9950e3ba-5d5c-480d-bbc7-9d37cedb3222" />

**Additional context**
Provide an accessible name for the link via the contents to explain the functionality of this link.

**Comment by sherenemorris03** [comment]
Issue fixed and retested by Jake our internal auditor. No further action required
